lib/maze.rb

```diff
@@ -8,7 +8,7 @@
 # providing an alternative to the proliferation of global variables or singletons.
 module Maze
 
-  VERSION = '9.35.3'
+  VERSION = '9.35.4'
```

Comment:
Minor version bump for enhancements, even is they seem small.
```suggestion
  VERSION = '9.36.0'
```